include/libexfat.h

```diff
@@ -90,6 +97,16 @@ off_t exfat_get_root_entry_offset(struct exfat_blk_dev *bd);
 int exfat_get_volume_label(struct exfat_blk_dev *bd, off_t root_clu_off);
 int exfat_set_volume_label(struct exfat_blk_dev *bd,
 		char *label_input, off_t root_clu_off);
+int exfat_read_sector(struct exfat_blk_dev *bd, void *buf,
+		unsigned int sec_off);
+int exfat_write_sector(struct exfat_blk_dev *bd, void *buf,
+		unsigned int sec_off);
+int exfat_write_checksum_sector(struct exfat_blk_dev *bd,
+		unsigned int checksum, bool is_backup);
+int exfat_show_volume_serial(struct exfat_blk_dev *bd,
+		struct exfat_user_input *ui);
+int exfat_change_volume_serial(struct exfat_blk_dev *bd,
```

Comment:
I think exfat_get/set_volume_serial is better, because of exfat_get/set_volume_label.

Reply:
Okay. Will change it.

Reply:
Ah, checked more. exfat_show_volume_serial function name is correct. because this function will print volume serial value.
So I will change like this.
- exfat_show_volume_serial
- exfat_set_volume_serial
- exfat_show_volume_label
- exfat_set_volume_label

How about this ?